js/index.js

```diff
@@ -1,6 +1,46 @@
-// Iteration 1: Names and Input
-//
-// Iteration 2: Conditionals
+//iteration 1
 
+const hacker1 = "Pieter"; // tested this with other names so can be different from pauline
+const hacker2 = "John";
+console.log(`The driver's name is ${hacker1}`);
+console.log(`The navigator's name is ${hacker2}`);
 
-// Iteration 3: Loops
+//iteration 2
+
+if (hacker1.length > hacker2.length) {
+  console.log(
+    `The driver has the longest name, it has ${hacker1.length} characters`
+  );
+} else if (hacker1.length < hacker2.length) {
+  console.log(
+    `It seems that the navigator has the longest name, it has ${hacker2.length} characters.`
+  );
+} else {
+  console.log(
+    `Wow, you both have equally long names, ${hacker1.length} characters`
+  );
+}
+
+//iteration 3
+
+let driverCapitalized = "";
+let navigatorReversed = "";
+
+for (let i = 0; i < hacker1.length; i++) {
+  driverCapitalized += hacker1[i].toUpperCase() + " ";
+}
+console.log(driverCapitalized);
+
```

Comment:
here you could do this to avoid the last space:
```
let driverCapitalized = "";

for (let i = 0; i < hacker1.length; i++) {
  if (i < hacker1.length -1) {
    driverCapitalized += hacker1[i].toUpperCase() + " "
  }
  else {
    driverCapitalized += hacker1[i].toUpperCase();
  }
}
console.log(driverCapitalized);

```